paper/ms.tex

```diff
@@ -328,7 +331,8 @@ \section{Gaussian processes}\sectlabel{gps}
 be derived as the generalization of what we astronomers call ``$\chi^2$''
 to include the effects of correlated noise.
 The maximum likelihood values for the parameters $\bvec{\theta}$ and
-$\bvec{\alpha}$ for a given dataset $(\bvec{y},\,X)$ can be found by
+$\bvec{\alpha}$ for a given dataset $(\bvec{y},\,X)$ \todo{(Why
+isn't X boldface?)} can be found by
```

Comment:
I normally use lowercase bold for vectors and uppercase regular for matrices. Here X a matrix.